package.nls.de.json

```diff
@@ -6,6 +6,7 @@
   "commands.SAS.authorize": "Anmelden",
   "commands.SAS.close": "Schließen der aktuellen Verbindung",
   "commands.SAS.collapseAll": "Alles einklappen",
+  "commands.SAS.convertNotebookToFlow": "In Flow konvertieren...",
```

Comment:
Hey @sahbic. Can you revert this changes in this file for now? We have someone who is able to translate things into German, and having these left out will make it easier to see which ones to translate in the future (i.e. since our build process points these out)

Reply:
Should I also revert the changes to `l10n/bundle.l10n.de.json` ?

Reply:
Yes, please. Sorry I didn't mention that one

Reply:
done in 58a2508d839cc91690c8d15be92340dac3a29561